src/App.vue

```diff
@@ -1,21 +1,30 @@
 <template>
   <metainfo>
     <template v-slot:title="{ content }">{{
-      content ? `${content} | Plasm Apps Portal` : `Plasm Apps Portal`
+      content ? `${content} | Astar Apps Portal` : `Astar Apps Portal`
```

Comment:
When first loading the app, the default page title will be shown as `<title><%= htmlWebpackPlugin.options.title %></title>` which is `plasm-apps-vue`. Can you fix this so that we can set a default page metadata. For example, we can configure it so that the title will say `Astar Apps Portal` as the page title when it is still connecting to the chain.

Similarly, I think it's good to use the default network favicon for the local network as well.
![bandicam 2021-06-28 16-03-47-835](https://user-images.githubusercontent.com/40356749/123594283-79493900-d82a-11eb-8f92-ecd6b2afe811.jpg)